packages/sc-project-initializer/commands/init/package.json

```diff
@@ -25,8 +25,8 @@
         "@massalabs/as-types": "^0.0.3-dev",
         "@massalabs/massa-as-sdk": "^2.0.2-dev",
         "@massalabs/massa-sc-compiler": "^0.0.4-dev",
-        "@massalabs/massa-sc-deployer": "^0.0.4-dev",
-        "@massalabs/massa-web3": "^1.13.0",
+        "@massalabs/massa-sc-deployer": "../../../sc-deployer",
```

Comment:
keep it as it is, we will update the project initializer in a following MR

Reply:
done